database/migrations/0055-add_communities_table.ts

```diff
@@ -0,0 +1,28 @@
+/* MIGRATION TO CREATE THE FEDERATION COMMUNITY TABLES
+ *
+ * This migration creates the `community` and 'communityfederation' tables in the `apollo` database (`gradido_community`).
+ */
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export async function upgrade(queryFn: (query: string, values?: any[]) => Promise<Array<any>>) {
+  await queryFn(`
+    CREATE TABLE IF NOT EXISTS communities (
```

Comment:
I am for

```suggestion
    CREATE TABLE communities (
```

since this will fail if the table already exist - and we do expect it not to exist?!

Reply:
yes that's right. Is changed